.php-cs-fixer.dist.php

```diff
@@ -77,6 +77,10 @@
         'no_whitespace_before_comma_in_array' => ['after_heredoc' => true],
         'no_whitespace_in_blank_line' => true,
         'non_printable_character' => true,
+        'nullable_type_declaration' => [
+            'syntax' => 'question_mark',
```

Comment:
Just happened to see this change.

I would suggest against this, it doesn't play well with union types. E.g. `?int|float` is invalid. Just a thought.

Reply:
Hmm, but on the other hand union types are not supported before 8.0, meanwhile 8.4 shows a deprecation message for implicit null types. 🙄

Guess this is the only way then.